src/Intervals.jl

```diff
@@ -24,6 +24,7 @@ Base.eltype(::AbstractInterval{T}) where {T} = T
 Base.broadcastable(x::AbstractInterval) = Ref(x)
 bounds_types(x::AbstractInterval{T,L,R}) where {T,L,R} = (L, R)
 
+include("isfinite.jl")
```

Comment:
Would this be worthwhile to maybe export?

Reply:
I definitely don't want to export this as it'll cause conflicts with `Base.isfinite`